### ISSUE TITLE
Proper coloring for Rust example code

### ISSUE THREAD

**Issue by schmidsi** [graphprotocol/docs#450]
https://thegraph.com/docs/en/cookbook/substreams-powered-subgraphs/

<img width="1005" alt="Screenshot 2023-07-26 at 20 40 13" src="https://github.com/graphprotocol/docs/assets/103280/4e70d668-e16f-4528-b6f5-c679756feb30">

Not annotated
<img width="1199" alt="Screenshot 2023-07-26 at 20 40 33" src="https://github.com/graphprotocol/docs/assets/103280/689a21b0-0653-4b73-9e23-3f9d9b57d099">

**Comment by B2o5T** [comment]
Enabling rule to enforce code blocks have language will avoid it in the future

**Comment by benface** [comment]
@B2o5T – You worked on this in #358, right?

**Comment by B2o5T** [comment]
@benface Yes, I already worked on adding this rule, but my pr didn’t receive approvals
